helmfile/charts/notify-celery/values.yaml

```diff
@@ -193,10 +193,10 @@ nodes:
     service:
       serviceEnabled: true
   # CELERY FOR DELIVERY RECEIPTS
-  scalable-delivery:
-    name: delivery-scalable
-    newRelicAppName: "notification-celery-delivery-scalable-dev"
-    scriptArg: "sh /app/scripts/run_celery_delivery.sh"
+  scalable-receipts:
+    name: receipts-scalable
+    newRelicAppName: "notification-celery-receipts-scalable-dev"
+    scriptArg: "sh /app/scripts/run_celery_receipts.sh"
```

Comment:
We copy `run_celery_delivery.sh` to `run_celery_receipts.sh` [here](https://github.com/cds-snc/notification-api/pull/2418).